src/PKSim.Core/Model/PopulationSimulation.cs

```diff
@@ -279,5 +279,65 @@ public override void UpdateFromOriginalSimulation(Simulation originalSimulation)
          var sourcePopSimulation = originalSimulation as PopulationSimulation;
          sourcePopSimulation?.AdvancedParameters.Each(x => AddAdvancedParameter(x, generateRandomValues: true));
       }
+
+      private DataColumn aggregateDataColumns(IEnumerable<DataColumn> columns)
```

Comment:
Move this outside of the simulation. This is clearly some kind of an extension methods. We have quite a few to calculate "aggregates" stuff already